pyproject.toml

```diff
@@ -42,6 +42,7 @@ exclude = ["tests", "docs"]
 [tool.poetry.dependencies]
 python = ">=3.5"
 django = ">=2"
+dj_database_url = "^0.5.0"
```

Comment:
This is not a dependency for jazzmin, dependencies for the test app are declared in tests/test_app/requirements.txt

Reply:
Oh, that's so right..

Reply:
@farridav the problem is that, according to the docs, the only thing a new dev has to do is to do `poetry install`. This won't work 100%, as we import `dj_database` module right on the settings.. But obviously for development you don't need all the packages that are on `requirements.txt`